sql/opt/exec/execbuilder: TestExecBuild failed

SHA: https://github.com/cockroachdb/cockroach/commits/edc18a6fc4bb4fbf118f4fc6c98e5642dc6adae7

Parameters:
```
TAGS=
GOFLAGS=-parallel=4
```

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=TestExecBuild PKG=github.com/cockroachdb/cockroach/pkg/sql/opt/exec/execbuilder TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1632406&tab=buildLog

```
=== RUN   TestExecBuild
--- FAIL: TestExecBuild (36.40s)
    test_log_scope.go:77: test logs captured to: /tmp/logTestExecBuild765913186
    test_log_scope.go:58: use -show-logs to present logs inline
=== RUN   TestExecBuild/local/delete_range
=== PAUSE TestExecBuild/local/delete_range
=== CONT  TestExecBuild/local/delete_range
rewrote:
CREATE TABLE xyz (x INT8, y INT8, z INT8, INDEX (z, y), FAMILY (z), FAMILY (x, y));


--- done: testdata/orderby with config local: 83 tests, 0 failures
        --- FAIL: TestExecBuild/local/delete_range (0.37s)
            logic.go:1652: 
                 
                testdata/delete_range:14: SELECT operation, message FROM [SHOW KV TRACE FOR SESSION]
                WHERE message LIKE '%DelRange%' OR message LIKE '%sending batch%'
                expected:
                    dist sender send  r26: sending batch 1 Get to (n1,s1):1
                    flow              DelRange /Table/53/1 - /Table/53/2
                    dist sender send  r26: sending batch 1 DelRng to (n1,s1):1
                    flow              DelRange /Table/53/1/601/0 - /Table/53/2
                    dist sender send  r26: sending batch 1 DelRng to (n1,s1):1
                    dist sender send  r26: sending batch 1 EndTxn to (n1,s1):1
                    
                but found (query options: "") :
                    flow              DelRange /Table/53/1 - /Table/53/2
                    dist sender send  r26: sending batch 1 DelRng to (n1,s1):1
                    flow              DelRange /Table/53/1/601/0 - /Table/53/2
                    dist sender send  r26: sending batch 1 DelRng to (n1,s1):1
                    dist sender send  r26: sending batch 1 EndTxn to (n1,s1):1
                    
                
            logic.go:2196: 
                testdata/delete_range:25: error while processing
            logic.go:2196: testdata/delete_range:25: too many errors encountered, skipping the rest of the input
=== RUN   TestExecBuild/local
    --- FAIL: TestExecBuild/local (0.00s)

```

SHA: https://github.com/cockroachdb/cockroach/commits/9d697080f227324220c09d1b63da7bbf969133ac

Parameters:
```
TAGS=
GOFLAGS=-parallel=4
```

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=TestExecBuild PKG=github.com/cockroachdb/cockroach/pkg/sql/opt/exec/execbuilder TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1633852&tab=buildLog

```
=== RUN   TestExecBuild
--- FAIL: TestExecBuild (33.41s)
    test_log_scope.go:77: test logs captured to: /tmp/logTestExecBuild144665136
    test_log_scope.go:58: use -show-logs to present logs inline
=== RUN   TestExecBuild/local
    --- FAIL: TestExecBuild/local (0.01s)
=== RUN   TestExecBuild/local/delete_range
=== PAUSE TestExecBuild/local/delete_range
=== CONT  TestExecBuild/local/delete_range
rewrote:
CREATE TABLE xyz (x INT8 PRIMARY KEY, y INT8, z INT8, INDEX foo (z, y), FAMILY (z), FAMILY (y));


rewrote:
CREATE TABLE abcd (a INT8, b INT8, c INT8, d INT8 NOT NULL, PRIMARY KEY (a, b, c), UNIQUE (d, b), FAMILY (d));


rewrote:
CREATE TABLE computed (k INT8 PRIMARY KEY, a JSONB, b STRING AS (a->>'q') STORED, INDEX (b), FAMILY (a, b));


--- done: testdata/distinct with config local: 22 tests, 0 failures
        --- FAIL: TestExecBuild/local/delete_range (0.88s)
            logic.go:1652: 
                 
                testdata/delete_range:14: SELECT operation, message FROM [SHOW KV TRACE FOR SESSION]
                WHERE message LIKE '%DelRange%' OR message LIKE '%sending batch%'
                expected:
                    dist sender send  r26: sending batch 1 Get to (n1,s1):1
                    flow              DelRange /Table/53/1 - /Table/53/2
                    dist sender send  r26: sending batch 1 DelRng to (n1,s1):1
                    flow              DelRange /Table/53/1/601/0 - /Table/53/2
                    dist sender send  r26: sending batch 1 DelRng to (n1,s1):1
                    dist sender send  r26: sending batch 1 EndTxn to (n1,s1):1
                    
                but found (query options: "") :
                    flow              DelRange /Table/53/1 - /Table/53/2
                    dist sender send  r26: sending batch 1 DelRng to (n1,s1):1
                    flow              DelRange /Table/53/1/601/0 - /Table/53/2
                    dist sender send  r26: sending batch 1 DelRng to (n1,s1):1
                    dist sender send  r26: sending batch 1 EndTxn to (n1,s1):1
                    
                
            logic.go:2196: 
                testdata/delete_range:25: error while processing
            logic.go:2196: testdata/delete_range:25: too many errors encountered, skipping the rest of the input

```

[(sql/opt/exec/execbuilder).TestExecBuild failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1635614&tab=buildLog) on [unknown@734e357412dadafcd6084b8eab8e251e44e86b4a](https://github.com/cockroachdb/cockroach/commits/734e357412dadafcd6084b8eab8e251e44e86b4a):

```
=== RUN   TestExecBuild
--- FAIL: TestExecBuild (29.52s)
    test_log_scope.go:77: test logs captured to: /tmp/logTestExecBuild826296353
    test_log_scope.go:58: use -show-logs to present logs inline
=== RUN   TestExecBuild/local/delete_range
=== PAUSE TestExecBuild/local/delete_range
=== CONT  TestExecBuild/local/delete_range
rewrote:
CREATE TABLE xyz (x INT8, y INT8, z INT8, pk1 INT8, pk2 INT8, PRIMARY KEY (pk1, pk2), FAMILY (y), FAMILY (z, x));


rewrote:
CREATE TABLE abc (a STRING, b STRING, c STRING, PRIMARY KEY (a, b, c));


rewrote:
CREATE TABLE tt (x INT8, y INT8, INDEX a (x), INDEX b (y), FAMILY (x), FAMILY (y));


--- done: testdata/distinct_on with config local: 32 tests, 0 failures
        --- FAIL: TestExecBuild/local/delete_range (0.55s)
            logic.go:1652: 
                 
                testdata/delete_range:14: SELECT operation, message FROM [SHOW KV TRACE FOR SESSION]
                WHERE message LIKE '%DelRange%' OR message LIKE '%sending batch%'
                expected:
                    dist sender send  r26: sending batch 1 Get to (n1,s1):1
                    flow              DelRange /Table/53/1 - /Table/53/2
                    dist sender send  r26: sending batch 1 DelRng to (n1,s1):1
                    flow              DelRange /Table/53/1/601/0 - /Table/53/2
                    dist sender send  r26: sending batch 1 DelRng to (n1,s1):1
                    dist sender send  r26: sending batch 1 EndTxn to (n1,s1):1
                    
                but found (query options: "") :
                    flow              DelRange /Table/53/1 - /Table/53/2
                    dist sender send  r26: sending batch 1 DelRng to (n1,s1):1
                    flow              DelRange /Table/53/1/601/0 - /Table/53/2
                    dist sender send  r26: sending batch 1 DelRng to (n1,s1):1
                    dist sender send  r26: sending batch 1 EndTxn to (n1,s1):1
                    
                
            logic.go:2196: 
                testdata/delete_range:25: error while processing
            logic.go:2196: testdata/delete_range:25: too many errors encountered, skipping the rest of the input
=== RUN   TestExecBuild/local
    --- FAIL: TestExecBuild/local (0.00s)

```

<details><summary>details</summary>
<p>
Parameters:

- TAGS=
- GOFLAGS=-parallel=4

```
make stressrace TESTS=TestExecBuild PKG=./pkg/sql/opt/exec/execbuilder TESTTIMEOUT=5m STRESSFLAGS=-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub>
</p>
</details>

[(sql/opt/exec/execbuilder).TestExecBuild failed](https://viewLog.html?buildId=1637089&tab=buildLog) on [master@7d130b78e820981ca106448f3a030ef94a2f0abe](https://github.com/cockroachdb/cockroach/commits/7d130b78e820981ca106448f3a030ef94a2f0abe):

```
=== RUN   TestExecBuild
--- FAIL: TestExecBuild (35.30s)
    test_log_scope.go:77: test logs captured to: /tmp/logTestExecBuild857373052
    test_log_scope.go:58: use -show-logs to present logs inline
=== RUN   TestExecBuild/local/show_trace/replica
--- done: testdata/show_trace with config local: 37 tests, 1 failures
rewrote:
CREATE TABLE "dec" (d DECIMAL, v DECIMAL(3,1), PRIMARY KEY (d, v));


--- progress: testdata/select: 115 statements/queries
rewrote:
CREATE TABLE num (i INT8 NULL, UNIQUE (i), f FLOAT8 NULL, UNIQUE (f), d DECIMAL NULL, UNIQUE (d), n INTERVAL NULL, UNIQUE (n), FAMILY (d), FAMILY (f, n), FAMILY (i));


            --- FAIL: TestExecBuild/local/show_trace/replica (0.06s)
                logic.go:1652: 
                     
                    testdata/show_trace:295: SELECT DISTINCT node_id, store_id, replica_id
                      FROM [SHOW EXPERIMENTAL_REPLICA TRACE FOR SESSION]
                    expected:
                        node_id  store_id  replica_id
                        1        1         26
                        1        1         6
                        1        1         27
                        
                    but found (query options: "colnames") :
                        node_id  store_id  replica_id
                        1        1         26
                        1        1         27
                        
                    
=== RUN   TestExecBuild/local/show_trace
=== PAUSE TestExecBuild/local/show_trace
=== CONT  TestExecBuild/local/show_trace
--- done: testdata/upsert with config local: 41 tests, 0 failures
        --- FAIL: TestExecBuild/local/show_trace (5.34s)
=== RUN   TestExecBuild/local
    --- FAIL: TestExecBuild/local (0.00s)

```

<details><summary>details</summary>
<p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestExecBuild PKG=./pkg/sql/opt/exec/execbuilder TESTTIMEOUT=5m STRESSFLAGS=-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub>
</p>
</details>

[(sql/opt/exec/execbuilder).TestExecBuild failed](https://viewLog.html?buildId=1637188&tab=buildLog) on [master@06dd5b2669c7f1fb281a2c99681593514932edee](https://github.com/cockroachdb/cockroach/commits/06dd5b2669c7f1fb281a2c99681593514932edee):

```
=== RUN   TestExecBuild
--- FAIL: TestExecBuild (34.55s)
    test_log_scope.go:77: test logs captured to: /tmp/logTestExecBuild695564464
    test_log_scope.go:58: use -show-logs to present logs inline
=== RUN   TestExecBuild/local/show_trace
=== PAUSE TestExecBuild/local/show_trace
=== CONT  TestExecBuild/local/show_trace
rewrote:
CREATE TABLE abc (a INT8, b INT8, c INT8, INDEX (c) STORING (a, b), FAMILY (a, b), FAMILY (c));


rewrote:
CREATE TABLE t35364 (x DECIMAL(10) PRIMARY KEY CHECK (round(x) = x), y DECIMAL(10) DEFAULT (1.5), z DECIMAL(10) CHECK (z >= 7) AS ((x + y) + 2.5) STORED, FAMILY (z), FAMILY (y));


--- progress: testdata/select: 188 statements/queries
--- done: testdata/update with config local: 38 tests, 0 failures
        --- FAIL: TestExecBuild/local/show_trace (4.81s)
=== RUN   TestExecBuild/local
    --- FAIL: TestExecBuild/local (0.01s)
=== RUN   TestExecBuild/local/show_trace/replica
--- done: testdata/show_trace with config local: 37 tests, 1 failures
--- total progress: 1536 statements/queries
            --- FAIL: TestExecBuild/local/show_trace/replica (0.04s)
                logic.go:1652: 
                     
                    testdata/show_trace:295: SELECT DISTINCT node_id, store_id, replica_id
                      FROM [SHOW EXPERIMENTAL_REPLICA TRACE FOR SESSION]
                    expected:
                        node_id  store_id  replica_id
                        1        1         26
                        1        1         6
                        1        1         27
                        
                    but found (query options: "colnames") :
                        node_id  store_id  replica_id
                        1        1         26
                        1        1         27
                        
                    

```

<details><summary>details</summary>
<p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestExecBuild PKG=./pkg/sql/opt/exec/execbuilder TESTTIMEOUT=5m STRESSFLAGS=-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub>
</p>
</details>

[(sql/opt/exec/execbuilder).TestExecBuild failed](https://viewLog.html?buildId=1640124&tab=buildLog) on [master@e81faed54ee90bdfed1dddc63bb13d1ecf8806da](https://github.com/cockroachdb/cockroach/commits/e81faed54ee90bdfed1dddc63bb13d1ecf8806da):

```
=== RUN   TestExecBuild
--- FAIL: TestExecBuild (33.39s)
    test_log_scope.go:77: test logs captured to: /tmp/logTestExecBuild596587672
    test_log_scope.go:58: use -show-logs to present logs inline
=== RUN   TestExecBuild/local/show_trace/replica
            --- FAIL: TestExecBuild/local/show_trace/replica (0.08s)
                logic.go:1652: 
                     
                    testdata/show_trace:295: SELECT DISTINCT node_id, store_id, replica_id
                      FROM [SHOW EXPERIMENTAL_REPLICA TRACE FOR SESSION]
                    expected:
                        node_id  store_id  replica_id
                        1        1         26
                        1        1         6
                        1        1         29
                        
                    but found (query options: "colnames") :
                        node_id  store_id  replica_id
                        1        1         26
                        1        1         29
                        
                    
=== RUN   TestExecBuild/local/show_trace
=== PAUSE TestExecBuild/local/show_trace
=== CONT  TestExecBuild/local/show_trace
        --- FAIL: TestExecBuild/local/show_trace (6.28s)
=== RUN   TestExecBuild/local
    --- FAIL: TestExecBuild/local (0.00s)

```

<details><summary>details</summary>
<p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestExecBuild PKG=./pkg/sql/opt/exec/execbuilder TESTTIMEOUT=5m STRESSFLAGS=-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub>
</p>
</details>

[(sql/opt/exec/execbuilder).TestExecBuild failed](https://viewLog.html?buildId=1641145&tab=buildLog) on [master@9d43e69185e256927c626f2447dfb6f4aaa8f3bd](https://github.com/cockroachdb/cockroach/commits/9d43e69185e256927c626f2447dfb6f4aaa8f3bd):

```
=== RUN   TestExecBuild
--- FAIL: TestExecBuild (49.40s)
    test_log_scope.go:77: test logs captured to: /tmp/logTestExecBuild791512952
    test_log_scope.go:58: use -show-logs to present logs inline
=== RUN   TestExecBuild/local/show_trace/replica
--- done: testdata/distinct with config local: 22 tests, 0 failures
--- done: testdata/distsql_auto_mode with config local: 20 tests, 0 failures
--- done: testdata/distinct_on with config local: 32 tests, 0 failures
            --- FAIL: TestExecBuild/local/show_trace/replica (0.12s)
                logic.go:1652: 
                     
                    testdata/show_trace:295: SELECT DISTINCT node_id, store_id, replica_id
                      FROM [SHOW EXPERIMENTAL_REPLICA TRACE FOR SESSION]
                    expected:
                        node_id  store_id  replica_id
                        1        1         26
                        1        1         6
                        1        1         29
                        
                    but found (query options: "colnames") :
                        node_id  store_id  replica_id
                        1        1         26
                        1        1         29
                        
                    
=== RUN   TestExecBuild/local/show_trace
=== PAUSE TestExecBuild/local/show_trace
=== CONT  TestExecBuild/local/show_trace
        --- FAIL: TestExecBuild/local/show_trace (4.01s)
=== RUN   TestExecBuild/local
    --- FAIL: TestExecBuild/local (0.00s)

```

<details><summary>details</summary>
<p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestExecBuild PKG=./pkg/sql/opt/exec/execbuilder TESTTIMEOUT=5m STRESSFLAGS=-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub>
</p>
</details>

[(sql/opt/exec/execbuilder).TestExecBuild failed](https://viewLog.html?buildId=1641182&tab=buildLog) on [master@da513804d58dcf8a66fe3fb41622d567e52b52e1](https://github.com/cockroachdb/cockroach/commits/da513804d58dcf8a66fe3fb41622d567e52b52e1):

```
=== RUN   TestExecBuild
--- FAIL: TestExecBuild (39.78s)
    test_log_scope.go:77: test logs captured to: /tmp/logTestExecBuild419530687
    test_log_scope.go:58: use -show-logs to present logs inline
=== RUN   TestExecBuild/local/show_trace/replica
            --- FAIL: TestExecBuild/local/show_trace/replica (0.11s)
                logic.go:1652: 
                     
                    testdata/show_trace:295: SELECT DISTINCT node_id, store_id, replica_id
                      FROM [SHOW EXPERIMENTAL_REPLICA TRACE FOR SESSION]
                    expected:
                        node_id  store_id  replica_id
                        1        1         26
                        1        1         6
                        1        1         29
                        
                    but found (query options: "colnames") :
                        node_id  store_id  replica_id
                        1        1         26
                        1        1         29
                        
                    
=== RUN   TestExecBuild/local/show_trace
=== PAUSE TestExecBuild/local/show_trace
=== CONT  TestExecBuild/local/show_trace
rewrote:
CREATE TABLE xyz (x INT8 PRIMARY KEY, y INT8, z INT8, INDEX foo (z, y), FAMILY (y), FAMILY (z));


rewrote:
CREATE TABLE t (a INT8, b INT8, c INT8, d INT8, j JSONB, s STRING, FAMILY (c, d), FAMILY (j, b), FAMILY (a), FAMILY (s));


rewrote:
CREATE TABLE abcd (a INT8, b INT8, c INT8, d INT8 NOT NULL, PRIMARY KEY (a, b, c), UNIQUE (d, b), FAMILY (d));


--- done: testdata/delete_range with config local: 7 tests, 0 failures
--- done: testdata/distinct with config local: 22 tests, 0 failures
        --- FAIL: TestExecBuild/local/show_trace (7.07s)
=== RUN   TestExecBuild/local
    --- FAIL: TestExecBuild/local (0.00s)

```

<details><summary>details</summary>
<p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestExecBuild PKG=./pkg/sql/opt/exec/execbuilder TESTTIMEOUT=5m STRESSFLAGS=-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub>
</p>
</details>

[(sql/opt/exec/execbuilder).TestExecBuild failed](https://viewLog.html?buildId=1641304&tab=buildLog) on [master@299f2729c546b9a2444665c7ccd4b07e62bc84b5](https://github.com/cockroachdb/cockroach/commits/299f2729c546b9a2444665c7ccd4b07e62bc84b5):

```
=== RUN   TestExecBuild
--- FAIL: TestExecBuild (49.72s)
    test_log_scope.go:77: test logs captured to: /tmp/logTestExecBuild601211269
    test_log_scope.go:58: use -show-logs to present logs inline
=== RUN   TestExecBuild/local/delete_range
=== PAUSE TestExecBuild/local/delete_range
=== CONT  TestExecBuild/local/delete_range
--- done: testdata/explain_env with config local: 21 tests, 0 failures
        --- FAIL: TestExecBuild/local/delete_range (1.04s)
            logic.go:1652: 
                 
                testdata/delete_range:14: SELECT operation, message FROM [SHOW KV TRACE FOR SESSION]
                WHERE message LIKE '%DelRange%' OR message LIKE '%sending batch%'
                expected:
                    dist sender send  r26: sending batch 1 Get to (n1,s1):1
                    flow              DelRange /Table/53/1 - /Table/53/2
                    dist sender send  r28: sending batch 1 DelRng to (n1,s1):1
                    flow              DelRange /Table/53/1/601/0 - /Table/53/2
                    dist sender send  r28: sending batch 1 DelRng to (n1,s1):1
                    dist sender send  r28: sending batch 1 EndTxn to (n1,s1):1
                    
                but found (query options: "") :
                    flow              DelRange /Table/53/1 - /Table/53/2
                    dist sender send  r28: sending batch 1 DelRng to (n1,s1):1
                    flow              DelRange /Table/53/1/601/0 - /Table/53/2
                    dist sender send  r28: sending batch 1 DelRng to (n1,s1):1
                    dist sender send  r28: sending batch 1 EndTxn to (n1,s1):1
                    
                
            logic.go:2196: 
                testdata/delete_range:25: error while processing
            logic.go:2196: testdata/delete_range:25: too many errors encountered, skipping the rest of the input
=== RUN   TestExecBuild/local
    --- FAIL: TestExecBuild/local (0.03s)

```

<details><summary>details</summary>
<p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestExecBuild PKG=./pkg/sql/opt/exec/execbuilder TESTTIMEOUT=5m STRESSFLAGS=-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub>
</p>
</details>

[(sql/opt/exec/execbuilder).TestExecBuild failed](https://viewLog.html?buildId=1644054&tab=buildLog) on [master@3078ef8af2f797d77869af469964a320c28c395e](https://github.com/cockroachdb/cockroach/commits/3078ef8af2f797d77869af469964a320c28c395e):

```
                     
                    testdata/show_trace:295: SELECT DISTINCT node_id, store_id, replica_id
                      FROM [SHOW EXPERIMENTAL_REPLICA TRACE FOR SESSION]
                    expected:
                        node_id  store_id  replica_id
                        1        1         26
                        1        1         6
                        1        1         29
                        
                    but found (query options: "colnames") :
                        node_id  store_id  replica_id
                        1        1         26
                        1        1         29
                        
                    
=== RUN   TestExecBuild/local/show_trace
=== PAUSE TestExecBuild/local/show_trace
=== CONT  TestExecBuild/local/show_trace
rewrote:
CREATE TABLE s (k1 INT8, k2 INT8, v INT8, PRIMARY KEY (k1, k2), FAMILY (v));


rewrote:
CREATE TABLE level3 (k1 INT8, k2 INT8, k3 INT8, PRIMARY KEY (k1, k2, k3), CONSTRAINT fk2 FOREIGN KEY (k1) REFERENCES level2) INTERLEAVE IN PARENT level2 (k1);


rewrote:
CREATE TABLE level4 (k1 INT8, k2 INT8, k3 INT8, PRIMARY KEY (k1, k2, k3), CONSTRAINT fk3 FOREIGN KEY (k1, k2, k3) REFERENCES level3) INTERLEAVE IN PARENT level3 (k1, k2, k3);


rewrote:
CREATE TABLE insert_t (x INT8, v INT8, FAMILY (v), FAMILY (x));


rewrote:
CREATE TABLE select_t (x INT8, v INT8, FAMILY (v), FAMILY (x));


rewrote:
CREATE TABLE p1_1 (i INT8 PRIMARY KEY, s1 STRING, s2 STRING, d DECIMAL, FAMILY (s1, s2, d)) INTERLEAVE IN PARENT p2 (i);


rewrote:
CREATE TABLE mutation (x INT8, y INT8 NOT NULL DEFAULT (10), FAMILY (y), FAMILY (x));
INSERT INTO mutation VALUES (1, 1);


        --- FAIL: TestExecBuild/local/show_trace (6.12s)
=== RUN   TestExecBuild/local
    --- FAIL: TestExecBuild/local (0.00s)
```

<details><summary>Repro</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestExecBuild PKG=./pkg/sql/opt/exec/execbuilder TESTTIMEOUT=5m STRESSFLAGS=-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/opt/exec/execbuilder).TestExecBuild failed](https://viewLog.html?buildId=1644922&tab=buildLog) on [master@beb69e089b6eadc0fde6c92eb533b08d248938fe](https://github.com/cockroachdb/cockroach/commits/beb69e089b6eadc0fde6c92eb533b08d248938fe):

```
=== RUN   TestExecBuild
--- FAIL: TestExecBuild (31.41s)
    test_log_scope.go:77: test logs captured to: /tmp/logTestExecBuild732345960
    test_log_scope.go:58: use -show-logs to present logs inline
=== RUN   TestExecBuild/local/show_trace/replica
--- done: testdata/show_trace with config local: 37 tests, 1 failures
rewrote:
CREATE TABLE num (i INT8 NULL, UNIQUE (i), f FLOAT8 NULL, UNIQUE (f), d DECIMAL NULL, UNIQUE (d), n INTERVAL NULL, UNIQUE (n), FAMILY (d), FAMILY (f), FAMILY (i, n));


            --- FAIL: TestExecBuild/local/show_trace/replica (0.03s)
                logic.go:1652: 
                     
                    testdata/show_trace:295: SELECT DISTINCT node_id, store_id, replica_id
                      FROM [SHOW EXPERIMENTAL_REPLICA TRACE FOR SESSION]
                    expected:
                        node_id  store_id  replica_id
                        1        1         26
                        1        1         6
                        1        1         29
                        
                    but found (query options: "colnames") :
                        node_id  store_id  replica_id
                        1        1         26
                        1        1         29
                        
                    
=== RUN   TestExecBuild/local/show_trace
=== PAUSE TestExecBuild/local/show_trace
=== CONT  TestExecBuild/local/show_trace
--- done: testdata/distinct with config local: 22 tests, 0 failures
        --- FAIL: TestExecBuild/local/show_trace (5.52s)
=== RUN   TestExecBuild/local
    --- FAIL: TestExecBuild/local (0.00s)
```

<details><summary>Repro</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestExecBuild PKG=./pkg/sql/opt/exec/execbuilder TESTTIMEOUT=5m STRESSFLAGS=-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/opt/exec/execbuilder).TestExecBuild failed](https://viewLog.html?buildId=1647416&tab=buildLog) on [master@5e14e74d1f0b41658ece3d9e618503d7efb4ffb0](https://github.com/cockroachdb/cockroach/commits/5e14e74d1f0b41658ece3d9e618503d7efb4ffb0):

```
=== RUN   TestExecBuild
--- FAIL: TestExecBuild (33.84s)
    test_log_scope.go:77: test logs captured to: /tmp/logTestExecBuild502178198
    test_log_scope.go:58: use -show-logs to present logs inline
=== RUN   TestExecBuild/local/show_trace/replica
rewrote:
CREATE TABLE coll (a STRING COLLATE da, b INT8, c BOOL, PRIMARY KEY (a, b), INDEX (b, a) STORING (c), FAMILY (c));


--- done: testdata/show_trace with config local: 37 tests, 1 failures
--- total progress: 1289 statements/queries
rewrote:
CREATE TABLE computed (k INT8 PRIMARY KEY, a JSONB, b STRING AS (a->>'q') STORED, INDEX (b), FAMILY (a), FAMILY (b));


            --- FAIL: TestExecBuild/local/show_trace/replica (0.08s)
                logic.go:1652: 
                     
                    testdata/show_trace:295: SELECT DISTINCT node_id, store_id, replica_id
                      FROM [SHOW EXPERIMENTAL_REPLICA TRACE FOR SESSION]
                    expected:
                        node_id  store_id  replica_id
                        1        1         26
                        1        1         6
                        1        1         29
                        
                    but found (query options: "colnames") :
                        node_id  store_id  replica_id
                        1        1         26
                        1        1         29
                        
                    
=== RUN   TestExecBuild/local/show_trace
=== PAUSE TestExecBuild/local/show_trace
=== CONT  TestExecBuild/local/show_trace
rewrote:
CREATE TABLE abc (a INT8, b INT8, c INT8, INDEX (a, b DESC, c), FAMILY (c), FAMILY (b), FAMILY (a));


rewrote:
CREATE TABLE tab4 (col0 INT8, col1 FLOAT8, col3 INT8, col4 FLOAT8, FAMILY (col4, col1), FAMILY (col0, col3));


        --- FAIL: TestExecBuild/local/show_trace (6.33s)
=== RUN   TestExecBuild/local
    --- FAIL: TestExecBuild/local (0.01s)
```

<details><summary>Repro</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestExecBuild PKG=./pkg/sql/opt/exec/execbuilder TESTTIMEOUT=5m STRESSFLAGS=-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/opt/exec/execbuilder).TestExecBuild failed](https://viewLog.html?buildId=1648260&tab=buildLog) on [master@6f09eb301bee72d548acdb4ae8fface8bb3123c2](https://github.com/cockroachdb/cockroach/commits/6f09eb301bee72d548acdb4ae8fface8bb3123c2):

```
=== RUN   TestExecBuild
--- FAIL: TestExecBuild (35.52s)
    test_log_scope.go:77: test logs captured to: /tmp/logTestExecBuild766879687
    test_log_scope.go:58: use -show-logs to present logs inline
=== RUN   TestExecBuild/local/show_trace/replica
            --- FAIL: TestExecBuild/local/show_trace/replica (0.05s)
                logic.go:1652: 
                     
                    testdata/show_trace:295: SELECT DISTINCT node_id, store_id, replica_id
                      FROM [SHOW EXPERIMENTAL_REPLICA TRACE FOR SESSION]
                    expected:
                        node_id  store_id  replica_id
                        1        1         26
                        1        1         6
                        1        1         29
                        
                    but found (query options: "colnames") :
                        node_id  store_id  replica_id
                        1        1         26
                        1        1         29
                        
                    
=== RUN   TestExecBuild/local/show_trace
=== PAUSE TestExecBuild/local/show_trace
=== CONT  TestExecBuild/local/show_trace
--- done: testdata/union with config local: 11 tests, 0 failures
--- done: testdata/subquery_correlated with config local: 2 tests, 0 failures
rewrote:
CREATE TABLE abc (a INT8 PRIMARY KEY, b INT8, c INT8, FAMILY (c, b));


        --- FAIL: TestExecBuild/local/show_trace (7.30s)
=== RUN   TestExecBuild/local
    --- FAIL: TestExecBuild/local (0.02s)
```

<details><summary>Repro</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestExecBuild PKG=./pkg/sql/opt/exec/execbuilder TESTTIMEOUT=5m STRESSFLAGS=-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/opt/exec/execbuilder).TestExecBuild failed](https://viewLog.html?buildId=1648443&tab=buildLog) on [master@a7f4f4fb7c3a8b304ade764908f4178947f60b7a](https://github.com/cockroachdb/cockroach/commits/a7f4f4fb7c3a8b304ade764908f4178947f60b7a):

```
    test_log_scope.go:58: use -show-logs to present logs inline
=== RUN   TestExecBuild/local/delete_range
=== PAUSE TestExecBuild/local/delete_range
=== CONT  TestExecBuild/local/delete_range
rewrote:
CREATE TABLE abc (a INT8, b INT8, c INT8, INDEX (a, b DESC, c), FAMILY (a, c, b));


rewrote:
CREATE TABLE abc (a INT8, b INT8, c INT8, INDEX (c) STORING (a, b), FAMILY (a), FAMILY (b, c));


rewrote:
CREATE TABLE xyz (x INT8, y INT8, z INT8, pk1 INT8, pk2 INT8, PRIMARY KEY (pk1, pk2), FAMILY (y), FAMILY (z, x));


rewrote:
CREATE TABLE abc (a STRING, b STRING, c STRING, PRIMARY KEY (a, b, c));


rewrote:
CREATE TABLE t35364 (x DECIMAL(10) PRIMARY KEY CHECK (round(x) = x), y DECIMAL(10) DEFAULT (1.5), z DECIMAL(10) CHECK (z >= 7) AS ((x + y) + 2.5) STORED, FAMILY (y, z));


        --- FAIL: TestExecBuild/local/delete_range (1.36s)
            logic.go:1652: 
                 
                testdata/delete_range:14: SELECT operation, message FROM [SHOW KV TRACE FOR SESSION]
                WHERE message LIKE '%DelRange%' OR message LIKE '%sending batch%'
                expected:
                    dist sender send  r26: sending batch 1 Get to (n1,s1):1
                    flow              DelRange /Table/53/1 - /Table/53/2
                    dist sender send  r28: sending batch 1 DelRng to (n1,s1):1
                    flow              DelRange /Table/53/1/601/0 - /Table/53/2
                    dist sender send  r28: sending batch 1 DelRng to (n1,s1):1
                    dist sender send  r28: sending batch 1 EndTxn to (n1,s1):1
                    
                but found (query options: "") :
                    flow              DelRange /Table/53/1 - /Table/53/2
                    dist sender send  r28: sending batch 1 DelRng to (n1,s1):1
                    flow              DelRange /Table/53/1/601/0 - /Table/53/2
                    dist sender send  r28: sending batch 1 DelRng to (n1,s1):1
                    dist sender send  r28: sending batch 1 EndTxn to (n1,s1):1
                    
                
            logic.go:2196: 
                testdata/delete_range:25: error while processing
            logic.go:2196: testdata/delete_range:25: too many errors encountered, skipping the rest of the input
=== RUN   TestExecBuild/local
    --- FAIL: TestExecBuild/local (0.00s)
```

<details><summary>Repro</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestExecBuild PKG=./pkg/sql/opt/exec/execbuilder TESTTIMEOUT=5m STRESSFLAGS=-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/opt/exec/execbuilder).TestExecBuild failed](https://viewLog.html?buildId=1652228&tab=buildLog) on [master@d066d582cf9eea283a8447a1398d20e7182150da](https://github.com/cockroachdb/cockroach/commits/d066d582cf9eea283a8447a1398d20e7182150da):

```
=== RUN   TestExecBuild
--- FAIL: TestExecBuild (33.87s)
    test_log_scope.go:77: test logs captured to: /tmp/logTestExecBuild275971531
    test_log_scope.go:58: use -show-logs to present logs inline
=== RUN   TestExecBuild/local/delete_range
=== PAUSE TestExecBuild/local/delete_range
=== CONT  TestExecBuild/local/delete_range
rewrote:
CREATE TABLE zigzag (a INT8 PRIMARY KEY, b INT8, c FLOAT8, d FLOAT8, INDEX b_idx (b), INDEX c_idx (c), FAMILY (c), FAMILY (b, d));


--- done: testdata/check_constraints with config local: 3 tests, 0 failures
rewrote:
CREATE TABLE zigzag2 (a INT8, b INT8, c INT8, d INT8, CONSTRAINT a_b_idx UNIQUE (a, b), INDEX c_idx (c), FAMILY (b, a, d), FAMILY (c));


rewrote:
CREATE TABLE abcd (a INT8, b INT8, c INT8, d INT8 NOT NULL, PRIMARY KEY (a, b, c), UNIQUE (d, b), FAMILY (d));


        --- FAIL: TestExecBuild/local/delete_range (1.01s)
            logic.go:1652: 
                 
                testdata/delete_range:14: SELECT operation, message FROM [SHOW KV TRACE FOR SESSION]
                WHERE message LIKE '%DelRange%' OR message LIKE '%sending batch%'
                expected:
                    dist sender send  r26: sending batch 1 Get to (n1,s1):1
                    flow              DelRange /Table/53/1 - /Table/53/2
                    dist sender send  r28: sending batch 1 DelRng to (n1,s1):1
                    flow              DelRange /Table/53/1/601/0 - /Table/53/2
                    dist sender send  r28: sending batch 1 DelRng to (n1,s1):1
                    dist sender send  r28: sending batch 1 EndTxn to (n1,s1):1
                    
                but found (query options: "") :
                    flow              DelRange /Table/53/1 - /Table/53/2
                    dist sender send  r28: sending batch 1 DelRng to (n1,s1):1
                    flow              DelRange /Table/53/1/601/0 - /Table/53/2
                    dist sender send  r28: sending batch 1 DelRng to (n1,s1):1
                    dist sender send  r28: sending batch 1 EndTxn to (n1,s1):1
                    
                
            logic.go:2197: 
                testdata/delete_range:25: error while processing
            logic.go:2197: testdata/delete_range:25: too many errors encountered, skipping the rest of the input
=== RUN   TestExecBuild/local
    --- FAIL: TestExecBuild/local (0.00s)
```

<details><summary>Repro</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestExecBuild PKG=./pkg/sql/opt/exec/execbuilder TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/opt/exec/execbuilder).TestExecBuild failed](https://viewLog.html?buildId=1656388&tab=buildLog) on [master@8c8e2293315caf10d990fd79549eec00d10a223c](https://github.com/cockroachdb/cockroach/commits/8c8e2293315caf10d990fd79549eec00d10a223c):

```
=== RUN   TestExecBuild
--- FAIL: TestExecBuild (39.30s)
    test_log_scope.go:77: test logs captured to: /tmp/logTestExecBuild034677037
    test_log_scope.go:58: use -show-logs to present logs inline
=== RUN   TestExecBuild/local/delete_range
=== PAUSE TestExecBuild/local/delete_range
=== CONT  TestExecBuild/local/delete_range
        --- FAIL: TestExecBuild/local/delete_range (1.05s)
            logic.go:1652: 
                 
                testdata/delete_range:14: SELECT operation, message FROM [SHOW KV TRACE FOR SESSION]
                WHERE message LIKE '%DelRange%' OR message LIKE '%sending batch%'
                expected:
                    dist sender send  r26: sending batch 1 Get to (n1,s1):1
                    flow              DelRange /Table/53/1 - /Table/53/2
                    dist sender send  r28: sending batch 1 DelRng to (n1,s1):1
                    flow              DelRange /Table/53/1/601/0 - /Table/53/2
                    dist sender send  r28: sending batch 1 DelRng to (n1,s1):1
                    dist sender send  r28: sending batch 1 EndTxn to (n1,s1):1
                    
                but found (query options: "") :
                    flow              DelRange /Table/53/1 - /Table/53/2
                    dist sender send  r28: sending batch 1 DelRng to (n1,s1):1
                    flow              DelRange /Table/53/1/601/0 - /Table/53/2
                    dist sender send  r28: sending batch 1 DelRng to (n1,s1):1
                    dist sender send  r28: sending batch 1 EndTxn to (n1,s1):1
                    
                
            logic.go:2197: 
                testdata/delete_range:25: error while processing
            logic.go:2197: testdata/delete_range:25: too many errors encountered, skipping the rest of the input
=== RUN   TestExecBuild/local
    --- FAIL: TestExecBuild/local (0.00s)
```

<details><summary>Repro</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestExecBuild PKG=./pkg/sql/opt/exec/execbuilder TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/opt/exec/execbuilder).TestExecBuild failed](https://viewLog.html?buildId=1682925&tab=buildLog) on [master@413fd1e46c2936029458db495efa242acb1b7d52](https://github.com/cockroachdb/cockroach/commits/413fd1e46c2936029458db495efa242acb1b7d52):

```
rewrote:
CREATE TABLE doubleparent (p1 INT8, p2 INT8, other INT8, PRIMARY KEY (p1, p2), FAMILY (p1, p2), FAMILY (other));


rewrote:
CREATE TABLE doublechild (c INT8 PRIMARY KEY, p1 INT8, p2 INT8, FOREIGN KEY (p1, p2) REFERENCES doubleparent (p1, p2), FAMILY (p2), FAMILY (c, p1));


rewrote:
CREATE TABLE grandchild (g INT8 PRIMARY KEY, c INT8 NOT NULL REFERENCES child (c), FAMILY (c), FAMILY (g));


--- progress: testdata/show_trace: 23 statements/queries
rewrote:
CREATE TABLE grandchild2 (g INT8 PRIMARY KEY, c INT8 NOT NULL REFERENCES child (c), FAMILY (c), FAMILY (g));


rewrote:
CREATE TABLE self (x INT8 PRIMARY KEY, y INT8 NOT NULL REFERENCES self (x), FAMILY (y, x));


--- progress: testdata/select_index: 119 statements/queries
--- done: testdata/fk_opt with config local: 43 tests, 0 failures
--- total progress: 889 statements/queries
        --- FAIL: TestExecBuild/local/delete_range (0.92s)
            logic.go:1652: 
                 
                testdata/delete_range:14: SELECT operation, message FROM [SHOW KV TRACE FOR SESSION]
                WHERE message LIKE '%DelRange%' OR message LIKE '%sending batch%'
                expected:
                    dist sender send  r26: sending batch 1 Get to (n1,s1):1
                    flow              DelRange /Table/53/1 - /Table/53/2
                    dist sender send  r28: sending batch 1 DelRng to (n1,s1):1
                    flow              DelRange /Table/53/1/601/0 - /Table/53/2
                    dist sender send  r28: sending batch 1 DelRng to (n1,s1):1
                    dist sender send  r28: sending batch 1 EndTxn to (n1,s1):1
                    
                but found (query options: "") :
                    flow              DelRange /Table/53/1 - /Table/53/2
                    dist sender send  r28: sending batch 1 DelRng to (n1,s1):1
                    flow              DelRange /Table/53/1/601/0 - /Table/53/2
                    dist sender send  r28: sending batch 1 DelRng to (n1,s1):1
                    dist sender send  r28: sending batch 1 EndTxn to (n1,s1):1
                    
                
            logic.go:2198: 
                testdata/delete_range:25: error while processing
            logic.go:2198: testdata/delete_range:25: too many errors encountered, skipping the rest of the input
=== RUN   TestExecBuild/local
    --- FAIL: TestExecBuild/local (0.00s)
```

<details><summary>Repro</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestExecBuild PKG=./pkg/sql/opt/exec/execbuilder TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/opt/exec/execbuilder).TestExecBuild failed](https://viewLog.html?buildId=1684285&tab=buildLog) on [master@a7469fbbbac10982482438d948fc3666dcf4a6c2](https://github.com/cockroachdb/cockroach/commits/a7469fbbbac10982482438d948fc3666dcf4a6c2):

```


rewrote:
CREATE TABLE zigzag2 (a INT8, b INT8, c INT8, d INT8, CONSTRAINT a_b_idx UNIQUE (a, b), INDEX c_idx (c), FAMILY (b, a, d, c));


rewrote:
CREATE TABLE unindexed (k INT8 PRIMARY KEY, v INT8, FAMILY (v), FAMILY (k));


rewrote:
CREATE TABLE indexed (id INT8 PRIMARY KEY, value INT8, other INT8, INDEX (value), FAMILY (value), FAMILY (id, other));


rewrote:
CREATE TABLE grandchild2 (g INT8 PRIMARY KEY, c INT8 NOT NULL REFERENCES child (c), FAMILY (g, c));


rewrote:
CREATE TABLE kw (k INT8 PRIMARY KEY, w INT8, FAMILY (w), FAMILY (k));


--- done: testdata/join with config local: 98 tests, 0 failures
--- total progress: 400 statements/queries
        --- FAIL: TestExecBuild/local/delete_range (1.42s)
            logic.go:1652: 
                 
                testdata/delete_range:14: SELECT operation, message FROM [SHOW KV TRACE FOR SESSION]
                WHERE message LIKE '%DelRange%' OR message LIKE '%sending batch%'
                expected:
                    dist sender send  r26: sending batch 1 Get to (n1,s1):1
                    flow              DelRange /Table/53/1 - /Table/53/2
                    dist sender send  r28: sending batch 1 DelRng to (n1,s1):1
                    flow              DelRange /Table/53/1/601/0 - /Table/53/2
                    dist sender send  r28: sending batch 1 DelRng to (n1,s1):1
                    dist sender send  r28: sending batch 1 EndTxn to (n1,s1):1
                    
                but found (query options: "") :
                    flow              DelRange /Table/53/1 - /Table/53/2
                    dist sender send  r28: sending batch 1 DelRng to (n1,s1):1
                    flow              DelRange /Table/53/1/601/0 - /Table/53/2
                    dist sender send  r28: sending batch 1 DelRng to (n1,s1):1
                    dist sender send  r28: sending batch 1 EndTxn to (n1,s1):1
                    
                
            logic.go:2198: 
                testdata/delete_range:25: error while processing
            logic.go:2198: testdata/delete_range:25: too many errors encountered, skipping the rest of the input
=== RUN   TestExecBuild/local
    --- FAIL: TestExecBuild/local (0.01s)
```

<details><summary>Repro</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestExecBuild PKG=./pkg/sql/opt/exec/execbuilder TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/opt/exec/execbuilder).TestExecBuild failed](https://viewLog.html?buildId=1684574&tab=buildLog) on [master@e3e0c7cbc452dfa24ae6442455ae0610fda7562a](https://github.com/cockroachdb/cockroach/commits/e3e0c7cbc452dfa24ae6442455ae0610fda7562a):

```
CREATE TABLE multi_ref_parent_bc (b INT8, c INT8, PRIMARY KEY (b, c), other INT8, FAMILY (c), FAMILY (other), FAMILY (b));


rewrote:
CREATE TABLE multi_ref_child (k INT8 PRIMARY KEY, a INT8, b INT8, c INT8, CONSTRAINT fk1 FOREIGN KEY (a) REFERENCES multi_ref_parent_a (a), CONSTRAINT fk2 FOREIGN KEY (b, c) REFERENCES multi_ref_parent_bc (b, c), FAMILY (a), FAMILY (c, k, b));


rewrote:
CREATE TABLE str (k INT8 PRIMARY KEY, v STRING, INDEX (v), FAMILY (k, v));


rewrote:
CREATE TABLE xy (x INT8, y INT8, INDEX (y), FAMILY (x), FAMILY (y));


rewrote:
CREATE TABLE child2 (c INT8 PRIMARY KEY, p INT8 NOT NULL REFERENCES parent (other), FAMILY (c, p));


rewrote:
CREATE TABLE xyz (x INT8 PRIMARY KEY, y INT8, z INT8, INDEX foo (z, y), FAMILY (y), FAMILY (z), FAMILY (x));


--- done: testdata/fk with config local: 6 tests, 0 failures
        --- FAIL: TestExecBuild/local/delete_range (1.01s)
            logic.go:1652: 
                 
                testdata/delete_range:14: SELECT operation, message FROM [SHOW KV TRACE FOR SESSION]
                WHERE message LIKE '%DelRange%' OR message LIKE '%sending batch%'
                expected:
                    dist sender send  r26: sending batch 1 Get to (n1,s1):1
                    flow              DelRange /Table/53/1 - /Table/53/2
                    dist sender send  r28: sending batch 1 DelRng to (n1,s1):1
                    flow              DelRange /Table/53/1/601/0 - /Table/53/2
                    dist sender send  r28: sending batch 1 DelRng to (n1,s1):1
                    dist sender send  r28: sending batch 1 EndTxn to (n1,s1):1
                    
                but found (query options: "") :
                    flow              DelRange /Table/53/1 - /Table/53/2
                    dist sender send  r28: sending batch 1 DelRng to (n1,s1):1
                    flow              DelRange /Table/53/1/601/0 - /Table/53/2
                    dist sender send  r28: sending batch 1 DelRng to (n1,s1):1
                    dist sender send  r28: sending batch 1 EndTxn to (n1,s1):1
                    
                
            logic.go:2198: 
                testdata/delete_range:25: error while processing
            logic.go:2198: testdata/delete_range:25: too many errors encountered, skipping the rest of the input
=== RUN   TestExecBuild/local
    --- FAIL: TestExecBuild/local (0.00s)
```

<details><summary>Repro</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestExecBuild PKG=./pkg/sql/opt/exec/execbuilder TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/opt/exec/execbuilder).TestExecBuild failed](https://viewLog.html?buildId=1687901&tab=buildLog) on [master@3d98dae853edbde083cc23b60ad6696958909849](https://github.com/cockroachdb/cockroach/commits/3d98dae853edbde083cc23b60ad6696958909849):

```
=== RUN   TestExecBuild
--- FAIL: TestExecBuild (48.13s)
    test_log_scope.go:77: test logs captured to: /tmp/logTestExecBuild745294081
    test_log_scope.go:58: use -show-logs to present logs inline
=== RUN   TestExecBuild/local/delete_range
=== PAUSE TestExecBuild/local/delete_range
=== CONT  TestExecBuild/local/delete_range
        --- FAIL: TestExecBuild/local/delete_range (0.51s)
            logic.go:1652: 
                 
                testdata/delete_range:14: SELECT operation, message FROM [SHOW KV TRACE FOR SESSION]
                WHERE message LIKE '%DelRange%' OR message LIKE '%sending batch%'
                expected:
                    dist sender send  r26: sending batch 1 Get to (n1,s1):1
                    flow              DelRange /Table/53/1 - /Table/53/2
                    dist sender send  r28: sending batch 1 DelRng to (n1,s1):1
                    flow              DelRange /Table/53/1/601/0 - /Table/53/2
                    dist sender send  r28: sending batch 1 DelRng to (n1,s1):1
                    dist sender send  r28: sending batch 1 EndTxn to (n1,s1):1
                    
                but found (query options: "") :
                    flow              DelRange /Table/53/1 - /Table/53/2
                    dist sender send  r28: sending batch 1 DelRng to (n1,s1):1
                    flow              DelRange /Table/53/1/601/0 - /Table/53/2
                    dist sender send  r28: sending batch 1 DelRng to (n1,s1):1
                    dist sender send  r28: sending batch 1 EndTxn to (n1,s1):1
                    
                
            logic.go:2198: 
                testdata/delete_range:25: error while processing
            logic.go:2198: testdata/delete_range:25: too many errors encountered, skipping the rest of the input
=== RUN   TestExecBuild/local
    --- FAIL: TestExecBuild/local (0.00s)
```

<details><summary>Repro</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestExecBuild PKG=./pkg/sql/opt/exec/execbuilder TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/opt/exec/execbuilder).TestExecBuild failed](https://viewLog.html?buildId=1695549&tab=buildLog) on [master@00c35450eb6dfdc6623f26420c1e065faf1ae4a9](https://github.com/cockroachdb/cockroach/commits/00c35450eb6dfdc6623f26420c1e065faf1ae4a9):

```
rewrote:
CREATE TABLE uvw (u INT8, v INT8, w INT8, INDEX (u, v, w), FAMILY (w), FAMILY (v, u));


rewrote:
CREATE TABLE uniontest (k INT8, v INT8, FAMILY (k), FAMILY (v));


rewrote:
CREATE TABLE kv (k INT8 PRIMARY KEY, v INT8, CONSTRAINT foo UNIQUE (v), INDEX bar (k, v), FAMILY (v), FAMILY (k));


rewrote:
CREATE TABLE unindexed (k INT8 PRIMARY KEY, v INT8, FAMILY (k, v));


rewrote:
CREATE TABLE indexed (id INT8 PRIMARY KEY, value INT8, other INT8, INDEX (value), FAMILY (other, value), FAMILY (id));


rewrote:
CREATE TABLE abc (a INT8, b INT8, c INT8, FAMILY (c), FAMILY (a), FAMILY (b));


        --- FAIL: TestExecBuild/local/delete_range (1.02s)
            logic.go:1652: 
                 
                testdata/delete_range:14: SELECT operation, message FROM [SHOW KV TRACE FOR SESSION]
                WHERE message LIKE '%DelRange%' OR message LIKE '%sending batch%'
                expected:
                    dist sender send  r26: sending batch 1 Get to (n1,s1):1
                    flow              DelRange /Table/53/1 - /Table/53/2
                    dist sender send  r28: sending batch 1 DelRng to (n1,s1):1
                    flow              DelRange /Table/53/1/601/0 - /Table/53/2
                    dist sender send  r28: sending batch 1 DelRng to (n1,s1):1
                    dist sender send  r28: sending batch 1 EndTxn to (n1,s1):1
                    
                but found (query options: "") :
                    flow              DelRange /Table/53/1 - /Table/53/2
                    dist sender send  r28: sending batch 1 DelRng to (n1,s1):1
                    flow              DelRange /Table/53/1/601/0 - /Table/53/2
                    dist sender send  r28: sending batch 1 DelRng to (n1,s1):1
                    dist sender send  r28: sending batch 1 EndTxn to (n1,s1):1
                    
                
            logic.go:2205: 
                testdata/delete_range:25: error while processing
            logic.go:2205: testdata/delete_range:25: too many errors encountered, skipping the rest of the input
=== RUN   TestExecBuild/local
    --- FAIL: TestExecBuild/local (0.00s)
```

<details><summary>Repro</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestExecBuild PKG=./pkg/sql/opt/exec/execbuilder TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/opt/exec/execbuilder).TestExecBuild failed](https://viewLog.html?buildId=1710350&tab=buildLog) on [master@1b68ec9ca10bbb55117cc367568fe175f838b49f](https://github.com/cockroachdb/cockroach/commits/1b68ec9ca10bbb55117cc367568fe175f838b49f):

```
=== RUN   TestExecBuild
--- FAIL: TestExecBuild (52.17s)
    test_log_scope.go:77: test logs captured to: /tmp/logTestExecBuild891711780
    test_log_scope.go:58: use -show-logs to present logs inline
=== RUN   TestExecBuild/local/delete_range
=== PAUSE TestExecBuild/local/delete_range
=== CONT  TestExecBuild/local/delete_range
rewrote:
CREATE TABLE t (a INT8 PRIMARY KEY, b INT8, INDEX ab (a, b), FAMILY (b, a));


--- progress: testdata/select: 78 statements/queries
--- done: testdata/distinct_on with config local: 32 tests, 0 failures
        --- FAIL: TestExecBuild/local/delete_range (0.84s)
            logic.go:1663: 
                 
                testdata/delete_range:14: SELECT operation, message FROM [SHOW KV TRACE FOR SESSION]
                WHERE message LIKE '%DelRange%' OR message LIKE '%sending batch%'
                expected:
                    dist sender send  r26: sending batch 1 Get to (n1,s1):1
                    flow              DelRange /Table/53/1 - /Table/53/2
                    dist sender send  r28: sending batch 1 DelRng to (n1,s1):1
                    flow              DelRange /Table/53/1/601/0 - /Table/53/2
                    dist sender send  r28: sending batch 1 DelRng to (n1,s1):1
                    dist sender send  r28: sending batch 1 EndTxn to (n1,s1):1
                    
                but found (query options: "") :
                    flow              DelRange /Table/53/1 - /Table/53/2
                    dist sender send  r28: sending batch 1 DelRng to (n1,s1):1
                    flow              DelRange /Table/53/1/601/0 - /Table/53/2
                    dist sender send  r28: sending batch 1 DelRng to (n1,s1):1
                    dist sender send  r28: sending batch 1 EndTxn to (n1,s1):1
                    
                
            logic.go:2216: 
                testdata/delete_range:25: error while processing
            logic.go:2216: testdata/delete_range:25: too many errors encountered, skipping the rest of the input
=== RUN   TestExecBuild/local
    --- FAIL: TestExecBuild/local (0.01s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestExecBuild PKG=./pkg/sql/opt/exec/execbuilder TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestExecBuild.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/opt/exec/execbuilder).TestExecBuild failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1719448&tab=buildLog) on [master@11a129329e5b5bc8aa22e1a7db15c7d56d328885](https://github.com/cockroachdb/cockroach/commits/11a129329e5b5bc8aa22e1a7db15c7d56d328885):

```
                    testdata/show_trace:311: SELECT operation, regexp_replace(regexp_replace(regexp_replace(message, 'job_id:[1-9]\d*', 'job_id:...', 'g'), 'wall_time:\d+', 'wall_time:...'), 'drop_time:\d+', 'drop_time:...', 'g') as message
                      FROM [SHOW KV TRACE FOR SESSION]
                    WHERE (message LIKE 'querying next range%' OR message LIKE '%batch%')
                      AND message NOT LIKE '%SystemConfigSpan%'
                      AND message NOT LIKE '%PushTxn%'
                    expected:
                        dist sender send  querying next range at /NamespaceTable/30/1/0/0/"test"/4/1
                        dist sender send  r26: sending batch 1 Get to (n1,s1):1
                        dist sender send  querying next range at /NamespaceTable/30/1/52/0/"system"/4/1
                        dist sender send  r26: sending batch 1 Get to (n1,s1):1
                        dist sender send  querying next range at /Table/3/1/12/2/1
                        dist sender send  r6: sending batch 1 Get to (n1,s1):1
                        dist sender send  querying next range at /Table/3/1/1/2/1
                        dist sender send  r6: sending batch 1 Get to (n1,s1):1
                        dist sender send  querying next range at /Table/12/1
                        dist sender send  r8: sending batch 1 Scan to (n1,s1):1
                    but found (query options: "") :
                        dist sender send  querying next range at /NamespaceTable/30/1/52/0/"system"/4/1
                        dist sender send  r26: sending batch 1 Get to (n1,s1):1
                        dist sender send  querying next range at /Table/3/1/12/2/1
                        dist sender send  r6: sending batch 1 Get to (n1,s1):1
                        dist sender send  querying next range at /Table/3/1/1/2/1
                        dist sender send  r6: sending batch 1 Get to (n1,s1):1
                        dist sender send  querying next range at /Table/12/1
                        dist sender send  r8: sending batch 1 Scan to (n1,s1):1
                        
                    
=== RUN   TestExecBuild/local/show_trace
=== PAUSE TestExecBuild/local/show_trace
=== CONT  TestExecBuild/local/show_trace
rewrote:
CREATE TABLE abc (a INT8, b INT8, c INT8, INDEX (c) STORING (a, b), FAMILY (a, b), FAMILY (c));


rewrote:
CREATE TABLE t35364 (x DECIMAL(10) PRIMARY KEY CHECK (round(x) = x), y DECIMAL(10) DEFAULT (1.5), z DECIMAL(10) CHECK (z >= 7) AS ((x + y) + 2.5) STORED, FAMILY (z, y, x));


rewrote:
CREATE TABLE abc (a INT8, b INT8, c INT8, INDEX (a, b), FAMILY (a, c, b));


--- done: testdata/orderby with config local: 83 tests, 0 failures
rewrote:
CREATE TABLE t5 (k INT8 PRIMARY KEY, a INT8, b INT8 CHECK (a > b), FAMILY (b, k, a));


        --- FAIL: TestExecBuild/local/show_trace (6.63s)
=== RUN   TestExecBuild/local
    --- FAIL: TestExecBuild/local (0.00s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestExecBuild PKG=./pkg/sql/opt/exec/execbuilder TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestExecBuild.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/opt/exec/execbuilder).TestExecBuild failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1719707&tab=buildLog) on [master@4d428767374e54b0b2aab3f96b90a54241b91b22](https://github.com/cockroachdb/cockroach/commits/4d428767374e54b0b2aab3f96b90a54241b91b22):

```
                        dist sender send  querying next range at /NamespaceTable/30/1/52/0/"system"/4/1
                        dist sender send  r26: sending batch 1 Get to (n1,s1):1
                        dist sender send  querying next range at /Table/3/1/12/2/1
                        dist sender send  r6: sending batch 1 Get to (n1,s1):1
                        dist sender send  querying next range at /Table/3/1/1/2/1
                        dist sender send  r6: sending batch 1 Get to (n1,s1):1
                        dist sender send  querying next range at /Table/12/1
                        dist sender send  r8: sending batch 1 Scan to (n1,s1):1
                        
                    
=== RUN   TestExecBuild/local/show_trace
=== PAUSE TestExecBuild/local/show_trace
=== CONT  TestExecBuild/local/show_trace
rewrote:
CREATE TABLE abc (a INT8 PRIMARY KEY, b INT8, c INT8, FAMILY (b), FAMILY (c, a));


rewrote:
CREATE TABLE doubleparent (p1 INT8, p2 INT8, other INT8, PRIMARY KEY (p1, p2), FAMILY (p1, other, p2));


rewrote:
CREATE TABLE doublechild (c INT8 PRIMARY KEY, p1 INT8, p2 INT8, FOREIGN KEY (p1, p2) REFERENCES doubleparent (p1, p2), FAMILY (c), FAMILY (p1, p2));


rewrote:
CREATE TABLE tab4 (col0 INT8, col1 FLOAT8, col3 INT8, col4 FLOAT8, FAMILY (col0, col1, col3, col4));


rewrote:
CREATE TABLE zigzag (a INT8 PRIMARY KEY, b INT8, c FLOAT8, d FLOAT8, INDEX b_idx (b), INDEX c_idx (c), FAMILY (b), FAMILY (a), FAMILY (c), FAMILY (d));


rewrote:
CREATE TABLE grandchild (g INT8 PRIMARY KEY, c INT8 NOT NULL REFERENCES child (c), FAMILY (g, c));


rewrote:
CREATE TABLE grandchild2 (g INT8 PRIMARY KEY, c INT8 NOT NULL REFERENCES child (c), FAMILY (g, c));


--- progress: testdata/join: 89 statements/queries
rewrote:
CREATE TABLE zigzag2 (a INT8, b INT8, c INT8, d INT8, CONSTRAINT a_b_idx UNIQUE (a, b), INDEX c_idx (c), FAMILY (b), FAMILY (d, a), FAMILY (c));


--- done: testdata/interleaved with config local: 44 tests, 0 failures
        --- FAIL: TestExecBuild/local/show_trace (5.73s)
=== RUN   TestExecBuild/local
    --- FAIL: TestExecBuild/local (0.00s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestExecBuild PKG=./pkg/sql/opt/exec/execbuilder TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestExecBuild.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/opt/exec/execbuilder).TestExecBuild failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1747363&tab=buildLog) on [master@2739821b911d777fa2a927295d699b559360a802](https://github.com/cockroachdb/cockroach/commits/2739821b911d777fa2a927295d699b559360a802):

```
=== RUN   TestExecBuild
--- FAIL: TestExecBuild (41.21s)
    test_log_scope.go:77: test logs captured to: /tmp/logTestExecBuild137306586
    test_log_scope.go:58: use -show-logs to present logs inline
=== RUN   TestExecBuild/local/autocommit
=== PAUSE TestExecBuild/local/autocommit
=== CONT  TestExecBuild/local/autocommit
        --- FAIL: TestExecBuild/local/autocommit (2.79s)
            logic.go:1674: 
                 
                testdata/autocommit:637: SELECT operation, message FROM [SHOW KV TRACE FOR SESSION]
                WHERE message       LIKE '%sending batch%'
                  AND message   NOT LIKE '%PushTxn%'
                  AND message   NOT LIKE '%QueryTxn%'
                  AND operation NOT LIKE '%async%'
                expected:
                    dist sender send  r28: sending batch 2 CPut, 2 InitPut to (n1,s1):1
                    dist sender send  r28: sending batch 2 Scan to (n1,s1):1
                    dist sender send  r28: sending batch 1 EndTxn to (n1,s1):1
                    
                but found (query options: "") :
                    dist sender send  r26: sending batch 1 Get to (n1,s1):1
                    dist sender send  r28: sending batch 2 CPut, 2 InitPut to (n1,s1):1
                    dist sender send  r28: sending batch 2 Scan to (n1,s1):1
                    dist sender send  r28: sending batch 1 EndTxn to (n1,s1):1
                    
                
            logic.go:2227: 
                testdata/autocommit:648: error while processing
            logic.go:2227: testdata/autocommit:648: too many errors encountered, skipping the rest of the input
=== RUN   TestExecBuild/local
    --- FAIL: TestExecBuild/local (0.00s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestExecBuild PKG=./pkg/sql/opt/exec/execbuilder TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestExecBuild.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/opt/exec/execbuilder).TestExecBuild failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1750148&tab=buildLog) on [master@e671a4ef97cbc6cf5d22f8f322fd45733d302094](https://github.com/cockroachdb/cockroach/commits/e671a4ef97cbc6cf5d22f8f322fd45733d302094):

```
=== PAUSE TestExecBuild/local/autocommit
=== CONT  TestExecBuild/local/autocommit
rewrote:
CREATE TABLE mutation (x INT8, y INT8 NOT NULL DEFAULT (10), FAMILY (y, x));
INSERT INTO mutation VALUES (1, 1);


rewrote:
CREATE TABLE abc (a INT8, b INT8, c INT8, INDEX (c) STORING (a, b), FAMILY (a, b), FAMILY (c));


rewrote:
CREATE TABLE xyz (x INT8, y INT8, z INT8, FAMILY (x, z), FAMILY (y));


rewrote:
CREATE TABLE t35364 (x DECIMAL(10) PRIMARY KEY CHECK (round(x) = x), y DECIMAL(10) DEFAULT (1.5), z DECIMAL(10) CHECK (z >= 7) AS ((x + y) + 2.5) STORED, FAMILY (z), FAMILY (y), FAMILY (x));


--- done: testdata/upsert with config local: 41 tests, 0 failures
--- done: testdata/insert with config local: 47 tests, 0 failures
rewrote:
CREATE TABLE t3 (k INT8 PRIMARY KEY, v INT8, w INT8, INDEX v (v), FAMILY (v, w), FAMILY (k));


        --- FAIL: TestExecBuild/local/autocommit (2.23s)
            logic.go:1674: 
                 
                testdata/autocommit:637: SELECT operation, message FROM [SHOW KV TRACE FOR SESSION]
                WHERE message       LIKE '%sending batch%'
                  AND message   NOT LIKE '%PushTxn%'
                  AND message   NOT LIKE '%QueryTxn%'
                  AND operation NOT LIKE '%async%'
                expected:
                    dist sender send  r28: sending batch 2 CPut, 2 InitPut to (n1,s1):1
                    dist sender send  r28: sending batch 2 Scan to (n1,s1):1
                    dist sender send  r28: sending batch 1 EndTxn to (n1,s1):1
                    
                but found (query options: "") :
                    dist sender send  r26: sending batch 1 Get to (n1,s1):1
                    dist sender send  r28: sending batch 2 CPut, 2 InitPut to (n1,s1):1
                    dist sender send  r28: sending batch 2 Scan to (n1,s1):1
                    dist sender send  r28: sending batch 1 EndTxn to (n1,s1):1
                    
                
            logic.go:2227: 
                testdata/autocommit:648: error while processing
            logic.go:2227: testdata/autocommit:648: too many errors encountered, skipping the rest of the input
=== RUN   TestExecBuild/local
    --- FAIL: TestExecBuild/local (0.00s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestExecBuild PKG=./pkg/sql/opt/exec/execbuilder TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestExecBuild.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/opt/exec/execbuilder).TestExecBuild failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1752964&tab=buildLog) on [master@1674c8aa4d955738365087d33ff707525e8a2b94](https://github.com/cockroachdb/cockroach/commits/1674c8aa4d955738365087d33ff707525e8a2b94):

```
=== RUN   TestExecBuild
--- FAIL: TestExecBuild (38.88s)
    test_log_scope.go:77: test logs captured to: /tmp/logTestExecBuild984716584
    test_log_scope.go:58: use -show-logs to present logs inline
=== RUN   TestExecBuild/local/autocommit
=== PAUSE TestExecBuild/local/autocommit
=== CONT  TestExecBuild/local/autocommit
rewrote:
CREATE TABLE xyz (x INT8, y INT8, z INT8, INDEX (z, y), FAMILY (x), FAMILY (y), FAMILY (z));


rewrote:
CREATE TABLE kv (k INT8 PRIMARY KEY, v INT8, CONSTRAINT foo UNIQUE (v), INDEX bar (k, v), FAMILY (v), FAMILY (k));


rewrote:
CREATE TABLE unindexed (k INT8 PRIMARY KEY, v INT8, FAMILY (k, v));


--- done: testdata/orderby with config local: 83 tests, 0 failures
rewrote:
CREATE TABLE indexed (id INT8 PRIMARY KEY, value INT8, other INT8, INDEX (value), FAMILY (id, other, value));


        --- FAIL: TestExecBuild/local/autocommit (2.38s)
            logic.go:1674: 
                 
                testdata/autocommit:637: SELECT operation, message FROM [SHOW KV TRACE FOR SESSION]
                WHERE message       LIKE '%sending batch%'
                  AND message   NOT LIKE '%PushTxn%'
                  AND message   NOT LIKE '%QueryTxn%'
                  AND operation NOT LIKE '%async%'
                expected:
                    dist sender send  r28: sending batch 2 CPut, 2 InitPut to (n1,s1):1
                    dist sender send  r28: sending batch 2 Scan to (n1,s1):1
                    dist sender send  r28: sending batch 1 EndTxn to (n1,s1):1
                    
                but found (query options: "") :
                    dist sender send  r26: sending batch 1 Get to (n1,s1):1
                    dist sender send  r28: sending batch 2 CPut, 2 InitPut to (n1,s1):1
                    dist sender send  r28: sending batch 2 Scan to (n1,s1):1
                    dist sender send  r28: sending batch 1 EndTxn to (n1,s1):1
                    
                
            logic.go:2227: 
                testdata/autocommit:648: error while processing
            logic.go:2227: testdata/autocommit:648: too many errors encountered, skipping the rest of the input
=== RUN   TestExecBuild/local
    --- FAIL: TestExecBuild/local (0.00s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestExecBuild PKG=./pkg/sql/opt/exec/execbuilder TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestExecBuild.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/opt/exec/execbuilder).TestExecBuild failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1769022&tab=buildLog) on [master@8448a07332397c7a02bbe472591a6a1002adccb9](https://github.com/cockroachdb/cockroach/commits/8448a07332397c7a02bbe472591a6a1002adccb9):

```
=== RUN   TestExecBuild
--- FAIL: TestExecBuild (38.49s)
    test_log_scope.go:77: test logs captured to: /tmp/logTestExecBuild537411329
    test_log_scope.go:58: use -show-logs to present logs inline
=== RUN   TestExecBuild/local/autocommit
=== PAUSE TestExecBuild/local/autocommit
=== CONT  TestExecBuild/local/autocommit
--- done: testdata/distinct_on with config local: 32 tests, 0 failures
        --- FAIL: TestExecBuild/local/autocommit (1.33s)
            logic.go:1674: 
                 
                testdata/autocommit:34: SELECT operation, message FROM [SHOW KV TRACE FOR SESSION]
                WHERE message     LIKE '%sending batch%'
                  AND message NOT LIKE '%PushTxn%'
                  AND message NOT LIKE '%QueryTxn%'
                expected:
                    dist sender send  r29: sending batch 1 CPut, 1 EndTxn to (n1,s1):1
                    
                but found (query options: "") :
                    dist sender send  r26: sending batch 1 Get to (n1,s1):1
                    dist sender send  r29: sending batch 1 CPut, 1 EndTxn to (n1,s1):1
                    
                
            logic.go:2227: 
                testdata/autocommit:42: error while processing
            logic.go:2227: testdata/autocommit:42: too many errors encountered, skipping the rest of the input
=== RUN   TestExecBuild/local
    --- FAIL: TestExecBuild/local (0.00s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestExecBuild PKG=./pkg/sql/opt/exec/execbuilder TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestExecBuild.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/opt/exec/execbuilder).TestExecBuild failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1794836&tab=buildLog) on [master@93b6a85dfee791e9fab68cb757591962308af208](https://github.com/cockroachdb/cockroach/commits/93b6a85dfee791e9fab68cb757591962308af208):

```
                    limit                                               ·                      ·
                     │                                                  count                  100
                     └── sort                                           ·                      ·
                          │                                             order                  -numwait,+s_name
                          └── group                                     ·                      ·
                               │                                        aggregate 0            s_name
                               │                                        aggregate 1            count_rows()
                               │                                        group by               s_name
                               └── render                               ·                      ·
                                    └── lookup-join                     ·                      ·
                                         │                              table                  orders@primary
                                         │                              type                   inner
                                         │                              equality               (l_orderkey) = (o_orderkey)
                                         │                              equality cols are key  ·
                                         │                              parallel               ·
                                         │                              pred                   @11 = 'F'
                                         └── lookup-join                ·                      ·
                                              │                         table                  nation@primary
                                              │                         type                   inner
                                              │                         equality               (s_nationkey) = (n_nationkey)
                                              │                         equality cols are key  ·
                                              │                         parallel               ·
                                              │                         pred                   @9 = 'SAUDI ARABIA'
                                              └── lookup-join           ·                      ·
                                                   │                    table                  supplier@primary
                                                   │                    type                   inner
                                                   │                    equality               (l_suppkey) = (s_suppkey)
                                                   │                    equality cols are key  ·
                                                   │                    parallel               ·
                                                   └── lookup-join      ·                      ·
                                                        │               table                  lineitem@primary
                                                        │               type                   semi
                                                        │               equality               (l_orderkey) = (l_orderkey)
                                                        │               pred                   @6 != @2
                                                        └── merge-join  ·                      ·
                                                             │          type                   anti
                                                             │          equality               (l_orderkey) = (l_orderkey)
                                                             │          mergeJoinOrder         +"(l_orderkey=l_orderkey)"
                                                             ├── scan   ·                      ·
                                                             │          table                  lineitem@primary
                                                             │          spans                  ALL
                                                             │          filter                 l_receiptdate > l_commitdate
                                                             └── scan   ·                      ·
                    ·                                                   table                  lineitem@primary
                    ·                                                   spans                  ALL
                    ·                                                   filter                 l_receiptdate > l_commitdate
                    
                
=== RUN   TestExecBuild/5node-dist
    --- FAIL: TestExecBuild/5node-dist (0.00s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestExecBuild PKG=./pkg/sql/opt/exec/execbuilder TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestExecBuild.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/opt/exec/execbuilder).TestExecBuild failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1794848&tab=buildLog) on [master@aafff65e69534ad92d7abc0fe4a2d8e5df5bfc77](https://github.com/cockroachdb/cockroach/commits/aafff65e69534ad92d7abc0fe4a2d8e5df5bfc77):

```
                    limit                                               ·                      ·
                     │                                                  count                  100
                     └── sort                                           ·                      ·
                          │                                             order                  -numwait,+s_name
                          └── group                                     ·                      ·
                               │                                        aggregate 0            s_name
                               │                                        aggregate 1            count_rows()
                               │                                        group by               s_name
                               └── render                               ·                      ·
                                    └── lookup-join                     ·                      ·
                                         │                              table                  orders@primary
                                         │                              type                   inner
                                         │                              equality               (l_orderkey) = (o_orderkey)
                                         │                              equality cols are key  ·
                                         │                              parallel               ·
                                         │                              pred                   @11 = 'F'
                                         └── lookup-join                ·                      ·
                                              │                         table                  nation@primary
                                              │                         type                   inner
                                              │                         equality               (s_nationkey) = (n_nationkey)
                                              │                         equality cols are key  ·
                                              │                         parallel               ·
                                              │                         pred                   @9 = 'SAUDI ARABIA'
                                              └── lookup-join           ·                      ·
                                                   │                    table                  supplier@primary
                                                   │                    type                   inner
                                                   │                    equality               (l_suppkey) = (s_suppkey)
                                                   │                    equality cols are key  ·
                                                   │                    parallel               ·
                                                   └── lookup-join      ·                      ·
                                                        │               table                  lineitem@primary
                                                        │               type                   semi
                                                        │               equality               (l_orderkey) = (l_orderkey)
                                                        │               pred                   @6 != @2
                                                        └── merge-join  ·                      ·
                                                             │          type                   anti
                                                             │          equality               (l_orderkey) = (l_orderkey)
                                                             │          mergeJoinOrder         +"(l_orderkey=l_orderkey)"
                                                             ├── scan   ·                      ·
                                                             │          table                  lineitem@primary
                                                             │          spans                  ALL
                                                             │          filter                 l_receiptdate > l_commitdate
                                                             └── scan   ·                      ·
                    ·                                                   table                  lineitem@primary
                    ·                                                   spans                  ALL
                    ·                                                   filter                 l_receiptdate > l_commitdate
                    
                
=== RUN   TestExecBuild/5node-dist
    --- FAIL: TestExecBuild/5node-dist (0.00s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestExecBuild PKG=./pkg/sql/opt/exec/execbuilder TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestExecBuild.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/opt/exec/execbuilder).TestExecBuild failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1794908&tab=buildLog) on [master@e7769aa272963f5bdc34cf41a2c406be09a91e92](https://github.com/cockroachdb/cockroach/commits/e7769aa272963f5bdc34cf41a2c406be09a91e92):

```
                    limit                                               ·                      ·
                     │                                                  count                  100
                     └── sort                                           ·                      ·
                          │                                             order                  -numwait,+s_name
                          └── group                                     ·                      ·
                               │                                        aggregate 0            s_name
                               │                                        aggregate 1            count_rows()
                               │                                        group by               s_name
                               └── render                               ·                      ·
                                    └── lookup-join                     ·                      ·
                                         │                              table                  orders@primary
                                         │                              type                   inner
                                         │                              equality               (l_orderkey) = (o_orderkey)
                                         │                              equality cols are key  ·
                                         │                              parallel               ·
                                         │                              pred                   @11 = 'F'
                                         └── lookup-join                ·                      ·
                                              │                         table                  nation@primary
                                              │                         type                   inner
                                              │                         equality               (s_nationkey) = (n_nationkey)
                                              │                         equality cols are key  ·
                                              │                         parallel               ·
                                              │                         pred                   @9 = 'SAUDI ARABIA'
                                              └── lookup-join           ·                      ·
                                                   │                    table                  supplier@primary
                                                   │                    type                   inner
                                                   │                    equality               (l_suppkey) = (s_suppkey)
                                                   │                    equality cols are key  ·
                                                   │                    parallel               ·
                                                   └── lookup-join      ·                      ·
                                                        │               table                  lineitem@primary
                                                        │               type                   semi
                                                        │               equality               (l_orderkey) = (l_orderkey)
                                                        │               pred                   @6 != @2
                                                        └── merge-join  ·                      ·
                                                             │          type                   anti
                                                             │          equality               (l_orderkey) = (l_orderkey)
                                                             │          mergeJoinOrder         +"(l_orderkey=l_orderkey)"
                                                             ├── scan   ·                      ·
                                                             │          table                  lineitem@primary
                                                             │          spans                  ALL
                                                             │          filter                 l_receiptdate > l_commitdate
                                                             └── scan   ·                      ·
                    ·                                                   table                  lineitem@primary
                    ·                                                   spans                  ALL
                    ·                                                   filter                 l_receiptdate > l_commitdate
                    
                
=== RUN   TestExecBuild/5node-dist
    --- FAIL: TestExecBuild/5node-dist (0.00s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestExecBuild PKG=./pkg/sql/opt/exec/execbuilder TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestExecBuild.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/opt/exec/execbuilder).TestExecBuild failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1794959&tab=buildLog) on [master@e7769aa272963f5bdc34cf41a2c406be09a91e92](https://github.com/cockroachdb/cockroach/commits/e7769aa272963f5bdc34cf41a2c406be09a91e92):

```
                    limit                                               ·                      ·
                     │                                                  count                  100
                     └── sort                                           ·                      ·
                          │                                             order                  -numwait,+s_name
                          └── group                                     ·                      ·
                               │                                        aggregate 0            s_name
                               │                                        aggregate 1            count_rows()
                               │                                        group by               s_name
                               └── render                               ·                      ·
                                    └── lookup-join                     ·                      ·
                                         │                              table                  orders@primary
                                         │                              type                   inner
                                         │                              equality               (l_orderkey) = (o_orderkey)
                                         │                              equality cols are key  ·
                                         │                              parallel               ·
                                         │                              pred                   @11 = 'F'
                                         └── lookup-join                ·                      ·
                                              │                         table                  nation@primary
                                              │                         type                   inner
                                              │                         equality               (s_nationkey) = (n_nationkey)
                                              │                         equality cols are key  ·
                                              │                         parallel               ·
                                              │                         pred                   @9 = 'SAUDI ARABIA'
                                              └── lookup-join           ·                      ·
                                                   │                    table                  supplier@primary
                                                   │                    type                   inner
                                                   │                    equality               (l_suppkey) = (s_suppkey)
                                                   │                    equality cols are key  ·
                                                   │                    parallel               ·
                                                   └── lookup-join      ·                      ·
                                                        │               table                  lineitem@primary
                                                        │               type                   semi
                                                        │               equality               (l_orderkey) = (l_orderkey)
                                                        │               pred                   @6 != @2
                                                        └── merge-join  ·                      ·
                                                             │          type                   anti
                                                             │          equality               (l_orderkey) = (l_orderkey)
                                                             │          mergeJoinOrder         +"(l_orderkey=l_orderkey)"
                                                             ├── scan   ·                      ·
                                                             │          table                  lineitem@primary
                                                             │          spans                  ALL
                                                             │          filter                 l_receiptdate > l_commitdate
                                                             └── scan   ·                      ·
                    ·                                                   table                  lineitem@primary
                    ·                                                   spans                  ALL
                    ·                                                   filter                 l_receiptdate > l_commitdate
                    
                
=== RUN   TestExecBuild/5node-dist
    --- FAIL: TestExecBuild/5node-dist (0.00s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestExecBuild PKG=./pkg/sql/opt/exec/execbuilder TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestExecBuild.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/opt/exec/execbuilder).TestExecBuild failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1794971&tab=buildLog) on [master@50b31b6ac2ac27f030869d5f076eb032f0029e73](https://github.com/cockroachdb/cockroach/commits/50b31b6ac2ac27f030869d5f076eb032f0029e73):

```
                    limit                                               ·                      ·
                     │                                                  count                  100
                     └── sort                                           ·                      ·
                          │                                             order                  -numwait,+s_name
                          └── group                                     ·                      ·
                               │                                        aggregate 0            s_name
                               │                                        aggregate 1            count_rows()
                               │                                        group by               s_name
                               └── render                               ·                      ·
                                    └── lookup-join                     ·                      ·
                                         │                              table                  orders@primary
                                         │                              type                   inner
                                         │                              equality               (l_orderkey) = (o_orderkey)
                                         │                              equality cols are key  ·
                                         │                              parallel               ·
                                         │                              pred                   @11 = 'F'
                                         └── lookup-join                ·                      ·
                                              │                         table                  nation@primary
                                              │                         type                   inner
                                              │                         equality               (s_nationkey) = (n_nationkey)
                                              │                         equality cols are key  ·
                                              │                         parallel               ·
                                              │                         pred                   @9 = 'SAUDI ARABIA'
                                              └── lookup-join           ·                      ·
                                                   │                    table                  supplier@primary
                                                   │                    type                   inner
                                                   │                    equality               (l_suppkey) = (s_suppkey)
                                                   │                    equality cols are key  ·
                                                   │                    parallel               ·
                                                   └── lookup-join      ·                      ·
                                                        │               table                  lineitem@primary
                                                        │               type                   semi
                                                        │               equality               (l_orderkey) = (l_orderkey)
                                                        │               pred                   @6 != @2
                                                        └── merge-join  ·                      ·
                                                             │          type                   anti
                                                             │          equality               (l_orderkey) = (l_orderkey)
                                                             │          mergeJoinOrder         +"(l_orderkey=l_orderkey)"
                                                             ├── scan   ·                      ·
                                                             │          table                  lineitem@primary
                                                             │          spans                  ALL
                                                             │          filter                 l_receiptdate > l_commitdate
                                                             └── scan   ·                      ·
                    ·                                                   table                  lineitem@primary
                    ·                                                   spans                  ALL
                    ·                                                   filter                 l_receiptdate > l_commitdate
                    
                
=== RUN   TestExecBuild/5node-dist
    --- FAIL: TestExecBuild/5node-dist (0.00s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestExecBuild PKG=./pkg/sql/opt/exec/execbuilder TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestExecBuild.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(sql/opt/exec/execbuilder).TestExecBuild failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1794983&tab=buildLog) on [master@eac401f1e69a2785d015807770eb59a4faa30486](https://github.com/cockroachdb/cockroach/commits/eac401f1e69a2785d015807770eb59a4faa30486):

```
                    limit                                               ·                      ·
                     │                                                  count                  100
                     └── sort                                           ·                      ·
                          │                                             order                  -numwait,+s_name
                          └── group                                     ·                      ·
                               │                                        aggregate 0            s_name
                               │                                        aggregate 1            count_rows()
                               │                                        group by               s_name
                               └── render                               ·                      ·
                                    └── lookup-join                     ·                      ·
                                         │                              table                  orders@primary
                                         │                              type                   inner
                                         │                              equality               (l_orderkey) = (o_orderkey)
                                         │                              equality cols are key  ·
                                         │                              parallel               ·
                                         │                              pred                   @11 = 'F'
                                         └── lookup-join                ·                      ·
                                              │                         table                  nation@primary
                                              │                         type                   inner
                                              │                         equality               (s_nationkey) = (n_nationkey)
                                              │                         equality cols are key  ·
                                              │                         parallel               ·
                                              │                         pred                   @9 = 'SAUDI ARABIA'
                                              └── lookup-join           ·                      ·
                                                   │                    table                  supplier@primary
                                                   │                    type                   inner
                                                   │                    equality               (l_suppkey) = (s_suppkey)
                                                   │                    equality cols are key  ·
                                                   │                    parallel               ·
                                                   └── lookup-join      ·                      ·
                                                        │               table                  lineitem@primary
                                                        │               type                   semi
                                                        │               equality               (l_orderkey) = (l_orderkey)
                                                        │               pred                   @6 != @2
                                                        └── merge-join  ·                      ·
                                                             │          type                   anti
                                                             │          equality               (l_orderkey) = (l_orderkey)
                                                             │          mergeJoinOrder         +"(l_orderkey=l_orderkey)"
                                                             ├── scan   ·                      ·
                                                             │          table                  lineitem@primary
                                                             │          spans                  ALL
                                                             │          filter                 l_receiptdate > l_commitdate
                                                             └── scan   ·                      ·
                    ·                                                   table                  lineitem@primary
                    ·                                                   spans                  ALL
                    ·                                                   filter                 l_receiptdate > l_commitdate
                    
                
=== RUN   TestExecBuild/5node-dist
    --- FAIL: TestExecBuild/5node-dist (0.00s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestExecBuild PKG=./pkg/sql/opt/exec/execbuilder TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestExecBuild.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

--- a/pkg/sql/opt/exec/execbuilder/testdata/lookup_join
+++ b/pkg/sql/opt/exec/execbuilder/testdata/lookup_join
@@ -1416,7 +1416,7 @@ ORDER BY numwait DESC, s_name
    LIMIT 100;
 ----
 ·                                                   distributed            true
-·                                                   vectorized             false
+·                                                   vectorized             true
 limit                                               ·                      ·
  │                                                  count                  100
  └── sort                                           ·                      ·

